extensions/mssql/src/objectExplorerNodeProvider/hdfsCommands.ts

```diff
@@ -45,14 +45,14 @@ function getSaveableUri(apiWrapper: ApiWrapper, fileName: string, isPreview?: bo
 	return vscode.Uri.file(fspath.join(root, fileName));
 }
 
-export async function getNode<T extends TreeNode>(context: ICommandViewContext |ICommandObjectExplorerContext, appContext: AppContext): Promise<T> {
+export async function getNode<T extends TreeNode>(sqlContext: ICommandViewContext | ICommandObjectExplorerContext, appContext: AppContext): Promise<T> {
```

Comment:
Why the name change? this isn't a sql-specific context, it's in fact a command context as indicated by the types. 

Can you please undo this change. In general, doing lots of changes like this make PRs harder as I need to see what's changed other than the naming. If we can, let's leave outside main PRs and do in "just cosmetic" PRs instead. That way the team can agree on naming.

Reply:
rolled back.

Reply:
it still shows as `sqlContext` to me, can you update again?